web/views/webclient_templates.xml

```diff
@@ -233,7 +233,7 @@
                         <t t-if="not disable_database_manager">
                             <a class="" href="/web/database/manager">Manage Databases</a> |
                         </t>
-                        <a href="https://www.odoo.com" target="_blank">Powered by <span>Odoo</span></a>
+                        <a href="https://www.odoo.com" target="_blank">Powered by <span>M-TIBA</span></a>
```

Comment:
This should be  rephrased to `"Powered by: Odoo, m-tiba"`. Also, need to get the url for m-tiba